classes/script_metadata.php

```diff
@@ -316,4 +316,15 @@ public static function get_sampling_period(): float {
         return round($insensiblerange ? $period : self::SAMPLING_PERIOD_DEFAUILT, 3);
     }
 
+    /**
+     * Returns the sampling double rate.
```

Comment:
A longer description here would be useful. What is this value, and what is it being used for? I see this "rate value" is being put into a limit field for the sample_set, but not sure how the values translate across.